pulp_ansible/app/galaxy/v3/views.py

```diff
@@ -45,7 +45,9 @@
 from pulp_ansible.app.viewsets import CollectionFilter, CollectionVersionFilter
 
 
-CollectionTuple = namedtuple("CollectionTuple", ["namespace", "name", "version", "pulp_created"])
+CollectionTuple = namedtuple(
+    "CollectionTuple", ["namespace", "name", "version", "pulp_created", "requires_ansible"]
```

Comment:
What motivated this change?

Reply:
https://github.com/pulp/pulp_ansible/pull/466/files#r567994874